### PR TITLE
Update jamied132.json

### PR DESCRIPTION
**Pull request by JamieD132**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] You have completed your website. 
- [X] The website is reachable.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/jamied132.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
     "email": "jamied132.email@gmail.com"
   },
   "record": {
-    "CNAME": "jamied132.github.io"
+    "CNAME": "jamied132.github.io",
+    "NS:["kevin.ns.cloudflare.com","megan.ns.cloudflare.com"]
```
Comment (CuteDog5695) [inline domains/jamied132.json:10]: You only need NS records.
```suggestion
    "NS":["kevin.ns.cloudflare.com","megan.ns.cloudflare.com"]
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Should be fine now, wait for the Admin to view your PR (copy and paste your reason here)

**Comment by JamieD132** [comment]
ok my reason (copied from #16898 ) is

This is my pull request to get an NS record. The reason are multiple reasons why I would like this:
 - I want to make a website where the main site is at jamied132.is-a.dev, and the api part of the site is api.jamied132.is-a.dev (because it looks more professional than api.jamied132.workers.dev)
 - At my school the workers.dev domain is blocked so i am incapable of editing/testing or even visiting the api or my actual site (because it relies on the api) while i am at school

**Comment by wdhdev** [comment]
I can relate to the school blocking thing 🤣

**Pull request merged**